src/mpi4py/util/dtlib.py

```diff
@@ -0,0 +1,276 @@
+# Author:  Lisandro Dalcin
+# Contact: dalcinl@gmail.com
+"""Convert NumPy and MPI datatypes."""
+# pylint: disable=too-many-locals
+# pylint: disable=too-many-branches
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-return-statements
+
+from .. import MPI
```

Comment:
Yes, but that's also the case with  `mpi4py.futures`, the new `mpi4py.util.pkl5`, and many other third-party packages that uses `mpi4py.MPI`. Remember we have the `mpi4py.rc` mechanism to avoid initialization. BTW, I recently added support for controlling `mpi4py.rc` via environment variables in e12723d (of course, I forgot to document it).

Reply:
I didn't know we accept env vars either! I can work on the doc later, I think we'll need a standalone page for env vars `MPI4PY_RC_*`.